src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+	<www-page
+		id="borrower-profile"
+		:header-theme="headerTheme"
+		:footer-theme="footerTheme"
```

Comment:
Since we are using the default themes for this page, you don't need to include the header-theme and footer-theme props. Removing them will also take care of these warnings in the console:
```
[Vue warn]: Invalid prop: type check failed for prop "headerTheme". Expected Object, got String with value "".
[Vue warn]: Invalid prop: type check failed for prop "footerTheme". Expected Object, got String with value "".
```